_includes/important.html

```diff
@@ -1,7 +1,6 @@
 <div markdown="1" class="alert alert-warning" role="alert">
   <i class="fa fa-warning"></i>
-  {% if include.title %}
-  <b>{{include.title}}</b>
-  {% else %}
-  <b>Important:</b>{% endif %} {{include.content}}
+  {% if include.title %}<b>{{include.title}}</b>
+  {% else %}<b>Important:</b>{% endif %}
+  {{include.content}}
 </div>
```

Comment:
I am using the same coding style as we have already in place for `note.html`, see here:

https://github.com/strongloop/loopback.io/blob/bfe2be34ed3148d08bd1b266eaaca75142a3b9d5/_includes/note.html#L1-L5